src/constant/generic.ts

```diff
@@ -8,9 +8,9 @@ import { Blockchain, BlockchainV2, EVMChain as EVMChainV2 } from "@carbon-sdk/ut
 export const DEFAULT_FEE_DENOM = "swth";
 
 // 0.0000001 SWTH
-// 10 SWTH sats
+// 15 SWTH sats
 export const DEFAULT_GAS_PRICE = new BigNumber(10);
-export const DEFAULT_GAS = new BigNumber(10000000);
+export const DEFAULT_GAS = new BigNumber(15000000);
```

Comment:
Hi, don't think this is enough. Cos the implementation is such that the SDK will take the default value from the [API call](https://api.carbon.network/carbon/fee/v1/gas_costs?pagination.limit=10000) (which is `10000000`), and only if the API call returns nothing, then we'll use `15000000`.
![image](https://github.com/user-attachments/assets/4f149f84-8a32-43b2-b9c0-b0f20f0a8f91)

Think you may need to find all submitted txs on Demex, and then override the gas cost for each of them.